src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -11,7 +18,21 @@ internal class CodeStylePage : AbstractOptionPage
     {
         protected override AbstractOptionPageControl CreateOptionPage(IServiceProvider serviceProvider)
         {
-            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s));
+            return new GridOptionPreviewControl(
+                serviceProvider,
+                (o, s) => new StyleViewModel(o, s),
+                GetEditorConfigOptions(),
+                LanguageNames.CSharp);
+        }
+
+        // internal for testing purposes.
+        internal static ImmutableArray<(string feature, ImmutableArray<IOption> options)> GetEditorConfigOptions()
```

Comment:
@sharwell has a pattern here that codifies the idea of "internal for testing purposes".

Reply:
Should stil use that pattern here.

Reply:
What is the pattern? Is there some attribute or naming?

---
In reply to: [219610046](https://github.com/dotnet/roslyn/pull/30043#discussion_r219610046) [](ancestors = 219610046)

Reply:
Ask @sharwell  :)  he has hte details.